core/src/main/java/com/intendia/reactivity/client/Gatekeeper.java

```diff
@@ -2,7 +2,17 @@
 
 @FunctionalInterface
 public interface Gatekeeper {
-    Gatekeeper PUBLIC = (PlaceRequest request) -> true;
+    Gatekeeper PUBLIC = (PlaceRequest request) -> {
+    };
 
-    boolean canReveal(PlaceRequest request);
+    default boolean canReveal(PlaceRequest request) {
```

Comment:
this looks weird 👎, althought it should support async evaluation, so we are going to apply this change, and we'll try to find a better API that supports async evaluation and more info if it cannot be revealed